### PR TITLE
Simplify website

### PR DESCRIPTION
**Pull request by avivkeller**
With the redesign of the main Node.js website, I think it's about time that this page is *slightly* updated. This PR makes the redirection automatic (w/ a `script` tag).

Also, it removes redundant files (E.G., the `index.html` file isn't needed because the `404.html` file has it's contents)

### CODE REVIEW

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
js.org does not allow for automatic redirects away from the domain. (We should probably add that to the README as you're not the first to open a PR to add such a change)

File: robots.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 User-agent: *
-Disallow: /
+Disallow: /
```
Comment (MattIPv4) [inline robots.txt]: newline eof, please?

File: 404.html [hunk 2]
```diff
@@ -1,120 +1,41 @@
...
-    </a></span>
-</main>
+  <div class="container">
+    <p>Redirecting you to <a href="https://nodejs.org/">nodejs.org</a>. If you are not redirected, <a
+        href="https://nodejs.org/">click here</a>.</p>
+  </div>
 </body>
-</html>
+
+</html>
```
Comment (MattIPv4) [inline 404.html]: Newline eof, please?

**Comment by avivkeller** [comment]
> js.org does not allow for automatic redirects away from the domain. (We should probably add that to the README as you're not the first to open a PR to add such a change)

Good to know! I've updated the redirection to use JS instead.

**Comment by MattIPv4** [comment]
🤔 The diff still shows that you've got logic in the file to automatically redirect the user?

**Comment by avivkeller** [comment]
> 🤔 The diff still shows that you've got logic in the file to automatically redirect the user?

Oops! Not anymore.

**Comment by MattIPv4** [comment]
I still see JS logic that is going to automatically redirect the user?

**Comment by avivkeller** [comment]
> I still see JS logic that is going to automatically redirect the user?

Yes, when you said that "js.org does not allow for automatic redirects away from the domain", weren't you referring to the `meta` tag, or no JS either?

**Comment by MattIPv4** [comment]
No JS either. You cannot automatically redirect the user away. It requires user interaction.

**Comment by avivkeller** [comment]
Ah, okay. Closing this, as it was the main point to add automatic redirects.

**Comment by MattIPv4** [comment]
👍 #8 was trying to do the same.

**Comment by avivkeller** [comment]
Thanks!
> Doing much of anything with node.js.org is (IMO) low-value work. I would leave it as it is. But if someone wants to do a little bit of a redesign on it to make it align more with our current design and content, that's fine. But it's also completely fine to leave it as it is. I doubt the site gets much traffic at all, and if it does, then it's doing the right thing by pointing users to the canonical site.

Maybe I'll take on this *little* task with a new PR. Thank you again! I love JS.org btw.

**Pull request closed without merging**